doc/spec/grammar/parser.y

```diff
@@ -531,6 +518,31 @@ argument    : expr
             | identifier '=' expr                  /* named arguments */
             ;
 
+/* parameters: separated by comma */
+
+parameters  : parameters1

```

Comment:
The `operation` production still uses this production (instead of `normalparams`), but it still rejects patterns in operation declarations, e.g. this is rejected:

```haskell
    with handler {
        fun yield(Cons(x,xs) : list<int>) { a.println }
    }
```

This is the behavior we want, but why is it working this way? Should I change `operation` to use `normalparams`?